src/lib/y2network/dialogs/popup.rb

```diff
@@ -52,7 +52,7 @@ def layout
         )
       end
 
-      protected
+    protected
```

Comment:
As there is already a CWM::Popup we should use it, it is already wrong in the branch I created.

https://github.com/yast/yast-yast2/blob/master/library/cwm/src/lib/cwm/popup.rb